#!/bin/bash
./manage.py runserver -v 3 --traceback 0.0.0.0:9000
